WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -84,6 +84,21 @@
   </div>
 </div>
 
+<!-- Hidden data range feature enabled by setting one of those parameters in the URL. -->
+<% if %i(date_range from_date to_date).any? { |key| params.key? key } %>
```

Comment:
It's a bit surprising. I don't know that it needs to change, but I also assumed that it's "`Date.parse` but safe" from the name.